dev-util/xdelta/xdelta-3.1.0.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A binary diff tool implementing VCDIFF"
+DESCRIPTION="Xdelta is a tool and library for differential compression. Xdelta \
+release version 3 supports VCDIFF encoding and decoding."
+HOMEPAGE="http://xdelta.org/"
+COPYRIGHT="2001-2015 Josh MacDonald"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/jmacd/xdelta/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="7515cf5378fca287a57f4e2fee1094aabc79569cfe60d91e06021a8fd7bae29d"
+SOURCE_DIR="xdelta-$portVersion/xdelta3"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	xdelta$secondaryArchSuffix = $portVersion
+	cmd:xdelta3 = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:liblzma$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	autoreconf --install
+	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir
+	make
+}
+
+INSTALL()
+{
+	make install
+}
+
+TEST()
+{
+	make check && xdelta3 test
```

Comment:
The recipe is OK, but **`xdelta3 test`** is failling on its 23rd test:
```
xdelta3: testing no_output...xdelta3: command was xdelta3 -q -f -e /tmp/xdtest.16470.target.3 /tmp/xdtest.16470.noperm.3
 failed: command should have not succeeded: XD3_INTERNAL
```
So if you could change TEST to:
```
TEST()
{
	make check
	xdelta3 test || true
}
```
that would be great.
I'll validate you task asap. Thanks again for your great work!
(I'll keep the PR unmerged for some hours in case someone finds a fix for the internal tests.)